worldedit-core/src/main/java/com/sk89q/worldedit/world/block/BaseBlock.java

```diff
@@ -188,11 +189,7 @@ public int hashCode() {
 
     @Override
     public String toString() {
-//        if (getNbtData() != null) { // TODO Maybe make some JSON serialiser to make this not awful.
-//            return blockState.getAsString() + " {" + String.valueOf(getNbtData()) + "}";
-//        } else {
-            return blockState.getAsString();
-//        }
+        return blockState.getAsString() + (hasNbtData() ? "{hasNbt}" : "");
```

Comment:
I feel the TODO is still worthwhile to have here. And pretty sure adventure (kyori text next version) includes an NBT to string serialiser anyway

Reply:
Okay, I'll restore it.